.code-samples.meilisearch.yaml

```diff
@@ -49,10 +49,32 @@ add_or_update_documents_1: |-
     -X PUT 'http://localhost:7700/indexes/movies/documents' \
     -H 'Content-Type: application/json' \
     --data-binary '[{
-        "id": 287947,
+        "reference_number": 287947,
```

Comment:
Why this one changed? Shouldn't it be `id`?

Reply:
The idea here is that it's an update of the document added in `add_or_update_documents_with_primaryKey_1`, so it has to have the same primary key `reference_number`.